app/js/features/signature/constants.js

```diff
@@ -1,4 +1,5 @@
 export const INCLUDE_SIGNATURE = 'INCLUDE_SIGNATURE';
+export const INCLUDE_SIGNATURE_PRE_SUCCESS = 'INCLUDE_SIGNATURE_PRE_SUCCESS';
 export const INCLUDE_SIGNATURE_SUCCEEDED = 'INCLUDE_SIGNATURE_SUCCEEDED';
```

Comment:
instead of PRE_SUCCESS and SUCCEEDED, maybe we should name them:
INCLUDE_SIGNATURE_TRANSACTION and
INCLUDE_SIGNATURE_RECEIPT